scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1592,6 +1592,10 @@
                 (str "Pod name from pod (" pod-name-from-pod ") "
                      "does not match pod name argument (" pod-name ")"))
         (log/info "In" compute-cluster-name "compute cluster, launching pod with name" pod-name "in namespace" namespace ":" (.serialize json pod))
+        (cook.config/log-passport-event {"pod-name" pod-name
+                                         "namespace" namespace
+                                         "cluster-name" compute-cluster-name
+                                         "event-type" "Launching Pod"})
```

Comment:
Any thoughts on keys being :symbols instead of "strings"?

Reply:
Maybe we should define all of the event-type values in a single place? Also, maybe something like `pod-launching` instead of `Launching Pod`?

Reply:
Would it make sense to add `instance-uuid` as a key for this event? I know it's the same as the pod name, but we probably want to be able to have a consistent key to tie things back to a job instance.

Reply:
> Any thoughts on keys being :symbols instead of "strings"?

Yeah, wasn't sure what the convention would be in this case. I've generally been using :symbols as the keys for all maps, but in this case since the map is created then immediately converted to a JSON string I figured strings would be appropriate. Open to discussing more though.

> Maybe we should define all of the event-type values in a single place? Also, maybe something like `pod-launching` instead of `Launching Pod`?

I like both of these changes-- the definitions will also have a good place to 'live' once I create the `cook.passport` namespace for the logging function.

> Would it make sense to add `instance-uuid` as a key for this event? I know it's the same as the pod name, but we probably want to be able to have a consistent key to tie things back to a job instance.

Yeah, I can add it. We'll also be iterating on what fields the messages have as we go along so any suggestions for info to include are welcome.

Reply:
> Would it make sense to add `instance-uuid` as a key for this event? I know it's the same as the pod name, but we probably want to be able to have a consistent key to tie things back to a job instance.

Are you saying to remove `pod-name` and just replace it with `instance-uuid`, or are you saying to keep both (even though they're the same)?

Reply:
> Are you saying to remove pod-name and just replace it with instance-uuid, or are you saying to keep both (even though they're the same)?

I don't feel too strongly, but keeping both seems reasonable to me.

Reply:
we don't have instance-uuid on a synthetic pod launch event. I would just keep pod name only. There will be other events that will need to be mapped to each other in a different layer.

Reply:
On a synthetic pod launch event, we don't have instance uuid, but we do have job uuid. 

I'd favor including either a `job-uuid` or `instance-uuid` key on every passport event.

What's the downside?

Reply:
The code would need to be refactored to pass in job-uuid and instance-uuid and it doesn't seen like there's a quick way to do it now. We also have a plan to have a passport query service that would have logic to join events. I want to spend some time designing that first and then finalizing the fields on all the events.

Reply:
Which code would need to be refactored?

Reply:
I don't see `job_uuid` or `instance_uuid` being passed into the `launch-pod` function, and it's not obviously seen in the `cook-expected-state-dict` map that gets passed into the function. I think the refactor @nsinkov mentioned would be refactoring to pass in the `job_uuid` and `instance_uuid` into the function.

Reply:
Ah good point, sorry I should have been more exact with my comment. I think that `job_uuid` not being passed in is the real issue. 

> we don't have instance-uuid on a synthetic pod launch event. I would just keep pod name only. There will be other events that will need to be mapped to each other in a different layer.

It relates back to the comment from @nsinkov I quoted, but for synthetic pods we can't use `pod-name` as the `instance_uuid`. So, if we want to include `job_uuid` or `instance_uuid` in every log, we would have to refactor `launch-pod` function to pass `job_uuid` into it in order to log `job_uuid` when synthetic pods are launched.

Reply:
Nikolai & I also talked a bit more about the passport query service a bit more after our end-user meeting yesterday and the plan is definitely for the passport query service to handle most/all of the logic for linking events. So at the moment I'm leaning towards just keeping `pod-name` as the field in this event.

Reply:
@dposada "The pod-name argument passed to launch-pod is the instance uuid" not for synthetic pods

Reply:
Yep, you're right, for synthetic pods, there is no relevant instance uuid, but we should be able to add a job-uuid field for synthetic pods.